host/generic/digitalpin.go

```diff
@@ -233,7 +234,7 @@ func (p *digitalPin) Close() error {
 	}
 
 	if err := p.drv.Unregister(p.id); err != nil {
-		return err
+		log.Println("Driver unregister error:", err.Error())
```

Comment:
libraries shouldn't log in general, why do you want to log instead of returning an error?